### PR TITLE
utils_test/libvirt: add check point for iface_check

### PR DESCRIPTION
**Pull request by PandaWei**
Add check point to check the state of specific interface.
return True if state is active, otherwise return False.

### CODE REVIEW

**Comment by PandaWei** [comment]
refer to autotest/tp-libvirt#510

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -1291,6 +1293,18 @@ def check_iface(iface_name, checkpoint, extra="", **dargs):
             iface_ip = iface.get_ip()
             check_pass = iface_ip == extra
             logging.debug("IP address of %s: %s", iface_name, iface_ip)
+        elif checkpoint == "state":
+            # check iface State
+            result = virsh.iface_list(extra, ignore_status=True)
+            check_exit_status(result, False)
+            output = re.findall(r"(\S+)\ +(\S+)\ +(\S+|\s+)[\ +\n]",
+                                str(result.stdout))
+            iface_state = filter(lambda x: x[0] == iface_name, output[1:])[0][1]
```
Comment (will-Do) [inline virttest/utils_test/libvirt.py]: It's better to deal with re and filter in a 'try...except' block

File: virttest/utils_test/libvirt.py [hunk 2]
```diff
@@ -1291,6 +1293,18 @@ def check_iface(iface_name, checkpoint, extra="", **dargs):
...
+            # active corresponds True, otherwise return False
+            if iface_state is not None and iface_state == "active":
+                check_pass = True
+            else:
+                check_pass = False
```
Comment (will-Do) [inline virttest/utils_test/libvirt.py]: ```check_pass = iface_state == "active"``` should be enough

Comment (PandaWei) [inline virttest/utils_test/libvirt.py]: Hi, 
Thanks for your comments.
I think it's very useful suggestion.
I will update this PR as it.

File: virttest/utils_test/libvirt.py (hunk 1 above)
Comment (PandaWei) [inline virttest/utils_test/libvirt.py]: check_iface() has a big try ... except that wraps up all of if and elif block.

Comment (will-Do) [inline virttest/utils_test/libvirt.py]: Thanks, then I'm good for this PR

**Comment by yangdongsheng** [comment]
Fine to me.

**Comment by Hao-Liu** [comment]
LGTM,  Merging. Thanks @PandaWei @will-Do @yangdongsheng .

**Pull request merged**